src/huggingface_hub/hf_api.py

```diff
@@ -670,9 +677,11 @@ def __init__(self, **kwargs):
                     blob_id=sibling.get("blobId"),
                     lfs=(
                         BlobLfsInfo(
-                            size=sibling["lfs"]["size"],
-                            sha256=sibling["lfs"]["sha256"],
-                            pointer_size=sibling["lfs"]["pointerSize"],
+                            {
+                                "size": sibling["lfs"]["size"],
+                                "sha256": sibling["lfs"]["sha256"],
+                                "pointer_size": sibling["lfs"]["pointerSize"],
+                            }
```

Comment:
Related to the comment above, this modification is not needed anymore.

```suggestion
                            size=sibling["lfs"]["size"],
                            sha256=sibling["lfs"]["sha256"],
                            pointer_size=sibling["lfs"]["pointerSize"],
```